arm-monitor/2017-04-01/swagger/activityLogAlerts_API.json

```diff
@@ -0,0 +1,451 @@
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActivityLogAlertNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "activityLogAlert",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ActivityLogAlertResource"
+                        },
+                        "description": "The activity log alert to create or use for the update."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An existing activity log alert was successfully updated.",
+                        "schema": {
+                            "$ref": "#/definitions/ActivityLogAlertResource"
+                        }
+                    },
+                    "201": {
+                        "description": "A new activity log alert was successfully created.",
+                        "schema": {
+                            "$ref": "#/definitions/ActivityLogAlertResource"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the activity log alert could not be created or updated.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "description": "Get an activity log alert.",
+                "operationId": "ActivityLogAlerts_Get",
+                "x-ms-examples": {
+                    "Get an activity log alert": {
+                        "$ref": "../examples/getActivityLogAlert.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActivityLogAlertNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/ActivityLogAlertResource"
```

Comment:
We likely only support patching tags (I'm not sure as I'm not very familiar with the activity log alerts apis), in which case the swagger checklist in the documentation specifies that the PATCH operation should not be exposed (see my other comment on the same for `ActionGroupResource`)